### PR TITLE
Improved Instructions for Linux and Minor change to Issue Template

### PR DESCRIPTION
**Pull request by VihagChaturvedi**
Hi, sorry for the late PR :P

Haven't had much time recently so I just managed to squeeze this in

Do need a little bit of revamp for Windows. Won't mind doing it if requested

Alternatively, this can be merged into the wiki as i believe it makes much more sense to put it in there

Also, I have added a small change to the Issue Template for bugs to give more details for Linux users, just to make it a bit more easier to troubleshoot

### CODE REVIEW

**Comment by bloc97** [comment]
Thanks, looks great! I also now think it might be better to eventually move the usage instructions to the wiki, maybe one instruction page for each OS as the instructions are starting to get heavy and might be hard to follow for novice users...

**Comment by bloc97** [comment]
In fact I just remembered why I did not put the install instructions to the wiki. It is not possible to link to older instructions in the releases section. Currently older releases have their instructions linking to the version branch, and using the wiki would not allow that (AFAIK), could be wrong. I think it would be better to just split the instructions into three files.

- Windows installation guide
- Linux installation guide
- Advanced usage guide (3rd part of the current installation instructions)

**Comment by VihagChaturvedi** [comment]
Or we can just copy paste this stuff in the wiki, i think that works too

**Comment by abhigit23** [comment]
Hey, could you do the same for the Flatpak version of mpv? I think flatpak is the easiest way to install mpv and it's available on almost all linux distros. You just need to put conf files and shaders to ~/.var/app/io.mpv.Mpv/mpv instead of ~/.config/mpv

like this:-
~/.var/app/io.mpv.Mpv/config/mpv/mpv.conf
~/.var/app/io.mpv.Mpv/config/mpv/input.conf
~/.var/app/io.mpv.Mpv/config/mpv/shaders
~/.var/app/io.mpv.Mpv/config/mpv/scripts

Reference from:-
https://flathub.org/apps/details/io.mpv.Mpv

I appreciate you added the guide for linux users too but if it's not too hassle to add Flatpak guide, you can think about it,  otherwise you can drop it.

**Comment by VihagChaturvedi** [comment]
> Hey, could you do the same for the Flatpak version of mpv? I think flatpak is the easiest way to install mpv and it's available on almost all linux distros. You just need to put conf files and shaders to ~/.var/app/io.mpv.Mpv/mpv instead of ~/.config/mpv
> 
> like this:- ~/.var/app/io.mpv.Mpv/config/mpv/mpv.conf ~/.var/app/io.mpv.Mpv/config/mpv/input.conf ~/.var/app/io.mpv.Mpv/config/mpv/shaders ~/.var/app/io.mpv.Mpv/config/mpv/scripts
> 
> Reference from:- https://flathub.org/apps/details/io.mpv.Mpv
> 
> I appreciate you added the guide for linux users too but if it's not too hassle to add Flatpak guide in the Linux section, you can think about it, otherwise you can ignore it.

Hey, thanks for pointing that out

It does make sense to add it, I will open a PR appending the information above, or you can do so as well

Thanks!

**Comment by abhigit23** [comment]
> Hey, thanks for pointing that out
> 
> It does make sense to add it, I will open a PR appending the information above, or you can do so as well
> 
> Thanks!

Hello, I have opened a PR, you can review it. If you find any problems let me know. It's literally my first PR here so please pardon if I miss something or lack in the quality.

Thanks.
